shipyard_blueprints/domo/pyproject.toml

```diff
@@ -1,22 +1,30 @@
 [tool.poetry]
 name = "shipyard-domo"
-version = "0.1.1"
+version = "0.2.0a0"
 description = "A local client for connecting and working with Domo"
 authors = ["wrp801 <wespoulsen@gmail.com>"]
 license = "Apache 2.0"
 readme = "README.md"
 packages = [{include = "shipyard_domo"}]
 
 [tool.poetry.dependencies]
-python = ">= 3.9, <3.10"
+python = "^3.9"
 requests = "2.31.0"
 pydomo = "^0.3.0.9"
-shipyard-templates = "0.5.0a0"
-pandas = "1.5.3"
-shipyard-utils = "0.1.4"
+shipyard-templates = "0.6.2"
+pandas = "^2.0"
+shipyard-bp-utils = "1.0.3"
 
 [tool.poetry.group.dev.dependencies]
-shipyard-templates = {path = "../../shipyard-templates"}
+pytest = "^8.0.2"
+python-dotenv = "^1.0.1"
+black = "^24.2.0"
+
+[tool.pytest.ini_options]
+testpaths = [
+    "tests",
+    "integration"
```

Comment:
is this needed? pytest by default looks for a tests folder

Reply:
I have needed to do that for my past two. For some reason it was defaulting to the root for me. When I added this in, it worked

Reply:
When I say the root, I mean the root of shipyard-blueprints